scripts/install

```diff
@@ -108,42 +118,34 @@ phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
 #=================================================
 # INSTALL LYCHEE WITH COMPOSER
 #=================================================
-ynh_script_progression --message="Installing Lychee with Composer..." --weight=15
+ynh_script_progression --message="Installing $app with Composer..." --weight=15
 
 ynh_install_composer --install_args="--ignore-platform-reqs"
 
 #=================================================
 # BUILDING
 #=================================================
-ynh_script_progression --message="Building..."
+ynh_script_progression --message="Building..." --weight=10
 
 # Setup application config
 ynh_add_config --template="../conf/.env.example" --destination="$final_path/.env"
-
+ 
 # Setup application config
 pushd $final_path
 	php$phpversion artisan key:generate -n --force --env
 	php$phpversion artisan migrate -n --force
 	php$phpversion artisan config:clear -n
 	php$phpversion artisan config:cache -n
+	patch -p1 < "$YNH_CWD/../conf/ffprobe.patch"
 popd
 
 # Setup custom user.css file
 ynh_add_config --template="../conf/user.css.example" --destination="$final_path/public/dist/user.css"
 
-#=================================================
-# SECURE FILES AND DIRECTORIES
-#=================================================
-
-# Set permissions to app files
-
-chmod 750 "$final_path"
-chmod 750 "$final_path/bootstrap/cache"
-chmod -R o-rwx "$final_path"
-chown -R $app:www-data "$final_path"
-
-chown -R $app: $public_path
-chmod 755 $public_path
+ynh_replace_string --match_string="\$ffmpeg = FFMpeg::create();" --replace_string="\$ffmpeg = FFMpeg::create(array(\
+        'ffmpeg.binaries'  => '/usr/bin/ffmpeg',\
+        'ffprobe.binaries' => '/usr/bin/ffprobe',\
+));" --target_file="$final_path/app/Actions/Photo/Extensions/VideoEditing.php"
```

Comment:
Ideally, sounds like ideally this could be handled with a patch rather than a sed

(But that's not super important if that works)

Reply:
I have tried very hard to do this with a patch, it failed miserably ._.